packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -425,8 +459,46 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, Semver {
             })
         );
 
-        // Set the L1 head hash at the time of the game's creation.
-        l1Head = Hash.wrap(blockhash(block.number - 1));
+        // Grab the index of the output proposal that commits to the starting L2 head.
+        // All outputs after this one are disputed.
+        // TODO(clabby): This is 2 calls too many for the information we need. Maybe
+        //               add a function to the L2OO?
```

Comment:
This should be tracked with a linear ticket where discussion can be had (in the future github issues - very important to not use close sourced linear software etc etc)

My 2c: it's not important enough to add logic to the L2OO - keep it plain and simple - minimal diff > small gas costs.